scripts/actions/flows.actions.js

```diff
@@ -34,6 +34,14 @@ export function selectColumn(columnIndex, columnId) {
   };
 }
 
+export function selectVectorLayers(vectorLayers, reloadLinks) {
+  return _getSelectAction('vectorLayers', vectorLayers, actions.SELECT_VECTOR_LAYERS, reloadLinks);
```

Comment:
When you toggle a vector layer, I'm not exactly sure where the data is gonna come from, but it'll be either:
- using data from the previous getNodes call
- or using a distinct API call
- or calling Carto SQL API

This data will not come from the get_links endpoint, so we should not use `_getSelectAction`. Just dispatch a  `SELECT_VECTOR_LAYERS` or `SELECT_CONTEXTUAL_LAYERS` action directly and we'll figure out later what should happen.